### ISSUE TITLE
The Max Results Value of The Algolia Search Action will be vanished after few minutes

### ISSUE THREAD

**Issue by ryutosh** [FlutterFlow/flutterflow-issues#3636]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

1. Make an action of The Algolia Search
2. Set Max Results to 20
3. Wait few minutes
4. gone..
![image](https://github.com/user-attachments/assets/fddc755d-1087-4550-8662-a2d7541b0e23)

### Expected Behavior

Hold value

### Steps to Reproduce

Same as current behavior

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITEehfKB35ZJpbwG1c7qcsJWiQMwJ0QlUIInj+0bWRMhfIioEZYuOuPBYlNLYtTneH43PE3+hmQywa30uPz9GMIEBxyWG9Q6ypRXQQrjWkm6WoyxPZagX3NQL/FYBWHE1sOnhSJDKsVsci0Z3zqACPCdG3qCf9qOYwx5e6fDbOY=

### Visual documentation

In Current Behavior section

### Environment

```markdown
- FlutterFlow version: 4.1.82
- Platform: MacOS
- Browser name and version: Chrome
- Operating system and version affected: Mac Siera
```

**Comment by ryutosh** [comment]
Maybe same problem with this.
https://github.com/FlutterFlow/flutterflow-issues/issues/2346#issuecomment-2273350830

**Comment by paulperez-dev** [comment]
Hi @ryutosh! It seems to be related. I will close this issue as duplicate.

If the fix did not solve this data loss specific issue, feel free to report it again.

**Comment by ryutosh** [comment]
Hi, @paulperez-dev 
The issue still present in ver. 4.1.91.
Not Fixed even if #2346 marked as closed.

https://github.com/user-attachments/assets/f6343e6e-2731-4b3e-bd6a-cd7b77f61d8b

**Comment by paulperez-dev** [comment]
Hi @ryutosh,

Could you please verify the behavior on version 5.0? If the issue persists, kindly create a new issue for further investigation.

Thank you!
